### PR TITLE
Improve Pkgs

### PR DESCRIPTION
**Pull request by erikarvstedt**
- Node-based pkgs: Use Node package lockfiles
- Add pkg metadata, as a follow-up to https://github.com/fort-nix/nix-bitcoin/pull/429

### CODE REVIEW

File: pkgs/spark-wallet/default.nix [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+let
+  nodePackages = import ./composition.nix { inherit pkgs; };
+in
+nodePackages.package.override {
+  # Required because spark-wallet uses `npm-shrinkwrap.json` as the lock file
+  reconstructLock = true;
+
+  meta = with lib; {
+    description = "A minimalistic wallet GUI for c-lightning";
+    homepage = "https://github.com/Ride-The-Lightning/RTL";
```
Comment (nixbitcoin) [inline pkgs/spark-wallet/default.nix:11]: https://github.com/shesek/spark-wallet

File: pkgs/secp256k1/default.nix [hunk 2]
```diff
@@ -21,6 +21,6 @@ stdenv.mkDerivation {
     homepage = "https://github.com/bitcoin-core/secp256k1";
     license = with licenses; [ mit ];
     maintainers = with maintainers; [ nixbitcoin ];
-    platforms = with platforms; unix;
+    platforms = platforms.unix;
```
Comment (nixbitcoin) [inline pkgs/secp256k1/default.nix:24]: Shouldn't the above lines also be edited in the same way?

Comment (erikarvstedt) [reply pkgs/secp256k1/default.nix:24]: Fixed for `license`, but keep the list for `maintainers`, which is a nixpkgs convention.

**Review by nixbitcoin — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

ACK 6f46651ed4bd2a38b3a80e2016b3748d21bb9c69

* Regenerated node packages from generate.sh
* Compared upstream versions/checksums to ones in node-packages.nix
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCgAdFiEEiDOlIkszv4ojhh+OtgROy6La5dAFAmG5+GkACgkQtgROy6La
5dA8hw//VZkOXQu8M9wvitjXsQVxU/b3vG+bcbd3TTKfP4ggynU/zAiMEh/bgrqm
YGOsW2RsY3d6Pf5VybojAZEXkjitF4MFWyEyCc4LICdEDBf1SoX1BQcljcTzRCFr
6Ktv6752xvTrOGfHPSEedOrmsljBoVEzjf+V0qhc2XrIhhAttwCPkouHjC2ehamI
y5ToOePpYcnpCfjJh9YR4S1UUA+L7vXJ8RSJjgXZQkgJ1oB7v0lPGXeZlFcHoRf6
728K23+d5U0JBikjNFeSg6NGzTn27oFGgNBSPV+Qfv88ifCwXpfbobE2FJMvfxB8
IIQgqHsX8IEdtDWdKxEZjuiCzU+uGWqhyYF9LHwpDwe0VtaFJvGljNDXQVXN9Zi+
fmeRGjbY6QD26hK57oHdBAkQvRu+7FUowbZrKlSsHl/FEANRgby9PPPeDieCm5SW
bZyrtgCILCu5BSb02OxRrCJP6oWGW7OE38mai9EgXeMj01+nTKwvs0BJPtgkLCC+
3kZJS9FuYYVrTmXp6+vX4ahqJ4GKw3fdZZLZLaQcu/ovT2KWQu2fRxsaGHdifm+i
uLvxEncKUbtSzY5G0D11/A2N2A1HBZJPzXWhE+oa8d8HkjL+NpL1j5hWViaTXfNO
AHYsI5Tj0cCo0mejrcKhB3T6woDow0dwpti0NSpKLboqsoHaPOA=
=yQ6D
-----END PGP SIGNATURE-----
```

**Review by nixbitcoin — APPROVED** [review]
```
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA512

reACK a31d07d8bf611a18247bfb85e3a47acbd60dc90a
-----BEGIN PGP SIGNATURE-----

iQIzBAEBCgAdFiEEiDOlIkszv4ojhh+OtgROy6La5dAFAmG/Q9wACgkQtgROy6La
5dAtwg/9GBd5oDO4jsybOWSllN57IO8Aa2b2FQJpNd0KuxjJNmgod0+9VLAa0QBK
UFSvHt6aScu+irqFrYLAdMhQgw2mlxULkdVK0K6TweRxVpHeqEvPh828/7TDc7KD
kDLjtJTJNcYsjfiztAYQQBDnjb11LoByFAw0nhJd7xCmjLDLtda7Nl3wC9YLxsw/
jiKsWmPMgE/2s82owhXgiZi+E0HQx9dDKES3fc9JL1MtA/Zwt9vzPfB0SkQj/63N
FYX5weXv6RWG/GEUlNJpN9AJEoUsg7spgTYePiBL33iFTaoDh/r124Jgcp0i77MO
2a9NqFhfu0NirmL0P1gkOnpWnlAz6Q7MtDtGplCdsAXV+KrqcKfSgvXpUxDiITX2
MF+s5n8THQjmPHXrIkfiA4xdv3Y/fPbYIGM/QZF+CCn4LzmGaU8t/Eft1qfQ5JNl
DC+oj/8rFIaf2Ciz85eOYU7WYSCD2BD1wC16DZQ/V2YOOHiysyy+gz5hQWzUeK6v
QipoDSNRjgihbgzW5yFkHOZNqISYc2SFe7OFH2+QuaAiXi6ZB7PzVDqvN2miwlAS
waQ2wk/tFVkj8mIYPlsm4B0wIaqy5nPvyW7M0A+CIhhPm72lLappuTC9fqw5PsgD
TNO0xR27iabyJ9sNYHdl9UFEofPilanQjjDn2Q5wb+TLE1tzbhs=
=Dd8U
-----END PGP SIGNATURE-----
```

**Pull request merged**